### PR TITLE
Adding new sensors : temperature, humidity, impulse count, CO2, VOC

### PR DESCRIPTION
**Pull request by Ben199-cpu**
#### Summary

Adding nke watteco's temperature and humidity sensors. Two impulse count sensors and two CO2 and VOC sensors and a pressure sensor

#### Changes

- ...
- ...

#### Notes for Reviewers

...

#### Release Notes

- ...

### CODE REVIEW

**Review by Jaime-Trinidad — APPROVED** [review]
files look fine

File: vendor/nke-watteco/th-sensor.yaml [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+    # LoRaWAN Device Profiles per region
+    # Supported regions are EU863-870, US902-928, AU915-928, AS923, CN779-787, EU433, CN470-510, KR920-923, IN865-867,
+    # RU864-870
+    profiles:
+      EU863-870:
+        # Optional identifier of the vendor of the profile. When you specify the vendorID, the profile is loaded from
+        # the vendorID's folder. This allows you to reuse profiles from module or LoRaWAN end device stack vendors.
+        # If vendorID is empty, the current vendor ID is used. In this example, the vendorID is the current vendor ID,
+        # which is verbose.
+        vendorID: nke-watteco
```
Comment (Jaime-Trinidad) [inline vendor/nke-watteco/th-sensor.yaml:37]: @Ben199-cpu please delete vendor ID from here

Comment (Ben199-cpu) [reply vendor/nke-watteco/th-sensor.yaml:37]: Ok, do I have to delete it from each sensors' file ? Should I do another push of these files afterwards ?

**Comment by Jaime-Trinidad** [comment]
@Ben199-cpu I clean files, please update branch and run make validate fmt

**Pull request merged**